src/renderers/native/ReactNative/ReactNative.js

```diff
@@ -15,61 +15,16 @@
 // the JS environment
```

Comment:
Ah, sorry. This got removed when I rebased on top of latest 15.0 changes in which React.js is already isomorphic. It wasn't before.